salt/utils/event.py

```diff
@@ -401,9 +422,10 @@ def _get_event(self, wait, tag, match_func=None):
                 if socks.get(self.sub) != zmq.POLLIN:
                     continue
 
-                # Please do not use non-blocking mode here. Reliability is
-                # more important than pure speed on the event bus.
-                ret = self.get_event_block()
+                if no_block is True:
```

Comment:
Did we before? This whole class is a bit of a mess IMO ;)
Before I don't think there was a specific "no_block" flag, I think we did just by setting wait to 0. IIRC zmq.Poller.poll(0) is non-blocking.